tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -36,11 +39,28 @@ export class RunsDataTable {
   @Input() headers!: ColumnHeader[];
   @Input() data!: TableData[];
   @Input() sortingInfo!: SortingInfo;
+  @Input() selectableColumns!: ColumnHeader[];
 
   ColumnHeaderType = ColumnHeaderType;
+  contextMenuHeader: ColumnHeader | undefined = undefined;
+  insertColumnTo: 'left' | 'right' | undefined = undefined;
```

Comment:
I think it's a little janky to have two seemingly unrelated properties here even though these properties are somewhat related. I can't think of a cleaner solution but I would love suggestions.

Reply:
This now reuses the Side enum